README.md

```diff
@@ -1,3 +1,7 @@
+# Slides From Project Fair, Spring 2024
+
+[Dominic Severe's Slides](https://docs.google.com/presentation/d/14ZIPUVmaSvwBqqNX1rT2Jot1E7lo8r0lFpHOoKVAPtE/edit#slide=id.g2e34ceb0e26_0_7)
```

Comment:
I don't usually approve of link integration but I think this is pretty safe and trust-able.